project_template/test/test_operations.cpp

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+#include <catch2/catch.hpp>
+#include <az/exampleshortname/operations.h>
+
+TEST_CASE("operations test", "[operations]") {

```

Comment:
>(" [](start = 9, length = 2)

Consider testing C code with a C framework.

This seems to be a C++ framework.

Reply:
yes it is. The testing frameworks, and their integration into tools (like cmake, visual studio, qt creator, eclipse, azure pipelines, github, jenkins, vim, emacs, etc is much better than any C testing framework I know about (closest would be mutest and gtest (gnome-test, not google test).

We don't consider it unreasonably to require a C++ compiler to build the tests.